package.json

```diff
@@ -35,13 +35,15 @@
     "react-native-tab-view": "^2.13.0",
     "react-native-vector-icons": "^6.6.0",
     "rn-fetch-blob": "^0.12.0",
-    "sendbird": "^3.0.119"
+    "sendbird": "^3.0.119",
+    "sinon": "^9.0.2"
   },
   "devDependencies": {
     "@babel/core": "^7.8.4",
     "@babel/runtime": "^7.8.4",
     "@react-native-community/eslint-config": "^0.0.7",
     "babel-jest": "^25.1.0",
+    "check-prop-types": "^1.1.2",
```

Comment:
Why is this dependency necessary? Sometimes errors say you need to download dependencies but that is rarely the case.

Reply:
We do need check-prop-types, it makes unit testing for prop-types easier.